mesa_geo/visualization/modules/MapVisualization.py

```diff
@@ -1,13 +1,41 @@
+from __future__ import annotations
+
+import dataclasses
+from dataclasses import dataclass
+from typing import Dict, Any
+
 import geopandas as gpd
 from folium.utilities import image_to_url
 from mesa.visualization.ModularVisualization import VisualizationElement
-from shapely.geometry import mapping
+from shapely.geometry import mapping, Point
 
 from mesa_geo.raster_layers import RasterLayer, RasterBase
 
 
+@dataclass
+class LeafletPortrayal:
+    """A dataclass defining the portrayal of a GeoAgent in Leaflet map.
+
+    The fields are defined to be consistent with GeoJSON options in
+    Leaflet.js: https://leafletjs.com/reference.html#geojson
+    """
+
+    style: Dict[str, Any] | None = None
```

Comment:
I think defining `str | bool | int | float` as a Mypy type is better than `Any`.

Reply:
Do you mean a type alias such as `LeafletOption = Union[str, bool, int, float]`?

Reply:
Yes, unfortunately it has to use `Union` for now.